pkg/awsutils/awsutils.go

```diff
@@ -155,6 +155,16 @@ type APIs interface {
 
 	// WaitForENIAndIPsAttached waits until the ENI has been attached and the secondary IPs have been added
 	WaitForENIAndIPsAttached(eni string, wantedSecondaryIPs int) (ENIMetadata, error)
+
+	//SetCNIunmanaged ENI
+	SetCNIUnmanagedENIs(eniID []string) error
```

Comment:
Another option to consider - Why can't we instead just enhance existing `SetUnmanagedENIs` and `IsUnmanagedENI` functions to also consider ENIs belonging to Non-Zero NW cards instead of managing them via separate functions? Might be easy to remove the additional conditions when we support them eventually - instead of the separate functions this CR utilizes?

Reply:
Yes I did think about that, but today any ENIs tagged with "no_manage" is stored in that cache so didn't want to deviate from that. Also in future if we have a use case where CNI needs to manage ENIs we can use the new cache or decommission once we have the support. Since we are planning for supporting this very soon, didn't want to modify existing routines.